.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+  schedule:
+    - cron: '0 1 * * *' # Test every day at 1AM
+  
+jobs:
+  build:
```

Comment:
Can we change this name to something more descriptive?  For example, "Full Analysis"?  @kmohrman @bryates or @Andrew42, might have a better suggestion.